playground/components/client.tsx

```diff
@@ -0,0 +1,6 @@
+'use client';
+import { Editor } from './editor';
+
+export default function ClientEntry() {
+  return <Editor />;
```

Comment:
I'm not sure we need a wrapper component. The intent seems to be to abstract the client component entry point. 

While this might sound like an easier mental load and organizing structure, it's adding a new layer of complexity to exports, imports and naming, making refactor difficult.

A flatter component structure will be much easier to refactor.